admin_guide/sdn_troubleshooting.adoc

```diff
@@ -576,16 +576,29 @@ NAT'd to the right port on the local machine for the `kubeproxy`.
 WARNING: This is all changing soon... Kubeproxy is being eliminated and
 replaced with an `iptables`-only solution.
 
-== Further Help
+[[sdn-troubleshooting-diagnostics-tool]]
+== Finding Network Issues Using the Diagnostics Tool
 
-. Run the script at
-https://raw.githubusercontent.com/openshift/openshift-sdn/master/hack/debug.sh
-on the master (or from another machine with access to the master) to
-generate useful debugging information.
-. When debugging IP failover problems, run the script at
+As a cluster administrator, run the diagnostics tool to diagnose common network
+issues:
+
+----
+# oadm diagnostics NetworkCheck
+----
+
+The diagnostics tool runs a series of checks for error conditions for the
+specified component. See the
+/admin_guide/diagnostics_tool.adoc#overview[Diagnostics Tool section] for more
+information.
+
+[NOTE]
+====
+Currently, the diagnostics tool cannot determine information about IP failover
```

Comment:
Hmm right. I didn't like that cos it says diagnostics tool a few words before, but it's better. Changed.